examples/templates/Azure/azure-pipelines-private-maven-repository.yml

```diff
@@ -0,0 +1,16 @@
+jobs:
+- job: debricked_scan
+  displayName: Debricked scan
+  pool:
+    vmImage: 'ubuntu-latest'
+  steps:
+    - task: MavenAuthenticate@0
```

Comment:
I would argue that this could go into the default template, with this task being commented out by default. With a comment above it like "uncomment the task below if you need to authenticate to a private Maven repository"

In order to reduce the amount of places to keep documentation up to date and in the end provide a complete example